stdlib/public/core/CMakeLists.txt

```diff
@@ -220,6 +222,10 @@ if(CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
       ${EXECINFO_LIBRARY})
 endif()
 
+if(CMAKE_SYSTEM_NAME STREQUAL "Windows")
+  list(APPEND swift_core_link_flags "$ENV{SystemRoot}/system32/bcrypt.dll")
+endif()
+
```

Comment:
@benrimmington Does this look correct?

Reply:
@Azoy, I think the `Bcrypt.lib` import library is needed.

<https://msdn.microsoft.com/en-us/library/windows/desktop/ms681914(v=vs.85).aspx>

Are you able to test on Windows? If not, shall we ask @compnerd for help?

Reply:
@compnerd, please see Azoy/swift@b2ef08f51de37a18d61d2c3869d116d0dc62dfe2 in Azoy/swift#2.

It uses `#pragma comment(lib, "Bcrypt.lib")` to add the import library.

It also uses `BCRYPT_SUCCESS` instead of `NT_SUCCESS`, so the extra defines/includes (`WIN32_NO_STATUS`, `<Ntstatus.h>`, `<Ntdef.h>`) aren't needed AFAIK.